pom.xml

```diff
@@ -374,7 +375,7 @@
 			<dependency>
 				<groupId>javax.mail</groupId>
 				<artifactId>mail</artifactId>
-				<version>1.4.1</version>
```

Comment:
Is this part of the ticket?

Reply:
yes it is @dkayiwa

Reply:
I have just seen this on the ticket. But am wondering why should these libraries be upgrade together in the same pull request? Are they related?

Reply:
I was following the ticket strictly. However, I can still separate them and do each library with it's pull request. What do you advise @dkayiwa

Reply:
I think the issue was about Javax, so all those lbrary under javax were grouped together in one ticket. @dkayiwa . Though I upgraded one at time until they all compiled locally then I made a joint PR.